src/libPMacc/include/particles/memory/boxes/ParticlesBox.hpp

```diff
@@ -55,13 +57,19 @@ class ParticlesBox : protected DataBox<PitchedBox<SuperCell<FRAME>, DIM> >
      * \warning after this call the object is in a invalid state and must be
      * initialized with an assignment of a valid ParticleBox
      */
-    HINLINE ParticlesBox()
+    HINLINE ParticlesBox() : hostMemoryOffset(0)
```

Comment:
I will change it to `HDINLINE` thus the constructor is equal to the other implementations